server/build.gradle

```diff
@@ -160,98 +159,46 @@ dependencies {
-    implementation "org.apache.johnzon:johnzon-core:1.2.2"
-
+    implementation libraries.hibernate
+    implementation libraries.ehcache
+    compileOnly "org.mozilla:jss"
+    testImplementation "org.mozilla:jss"
+    implementation "ldapjdk:ldapjdk"
+    implementation "org.quartz-scheduler:quartz"
+    implementation libraries.artemis
     // AMQP drivers for talking to QPID
-    implementation "org.apache.qpid:qpid-common:0.32"
-    implementation "org.apache.qpid:qpid-client:0.32"
-    implementation "org.apache.mina:mina-core:2.1.3"
-    implementation "org.apache.mina:mina-filter-ssl:1.1.7"
-    implementation "geronimo-spec:geronimo-spec-jms:1.1-rc4"
-
+    implementation libraries.amqp
     // Javascript engine
-    implementation "org.mozilla:rhino:1.7R3"
-
-    // Keycloak
-    compile group: 'org.keycloak', name: 'keycloak-servlet-filter-adapter', version: '7.0.0'
-    compile group: 'org.keycloak', name: 'keycloak-adapter-core', version: '7.0.0'
-    compile group: 'org.keycloak', name: 'keycloak-common', version: '7.0.0'
-    compile group: 'org.keycloak', name: 'keycloak-core', version: '7.0.0'
-    compile group: 'org.keycloak', name: 'keycloak-adapter-spi', version: '7.0.0'
-    compile group: 'org.keycloak', name: 'keycloak-servlet-adapter-spi', version: '7.0.0'
-    compile 'org.apache.httpcomponents:httpcore:4.4.7'
-    compile 'org.bouncycastle:bcpkix-jdk15on:1.60'
-    compile 'org.bouncycastle:bcprov-jdk15on:1.60'
-    compile group: 'org.keycloak', name: 'keycloak-jaxrs-oauth-client', version: '7.0.0'
-    compile group: 'org.keycloak', name: 'keycloak-authz-client', version: '7.0.0'
-
+    implementation "org.mozilla:rhino"
+    compile libraries.keycloak
     // Listed twice due to design decision by the Gradle team: https://discuss.gradle.org/t/compileonly-dependencies-are-not-available-in-tests/15366/3
     compileOnly libraries.javax_servlet
     testCompile libraries.javax_servlet
 
     if (use_logdriver) {
-        implementation "org.slf4j:log4j-over-slf4j:1.7.5"
-        implementation "logdriver:logdriver:1.0"
+        implementation "org.slf4j:log4j-over-slf4j"
+        implementation "logdriver:logdriver"
     }
 
     // DB Drivers
-    runtimeOnly "org.postgresql:postgresql:42.2.2"
-    runtimeOnly "mysql:mysql-connector-java:5.1.26"
-    runtimeOnly "org.mariadb.jdbc:mariadb-java-client:2.3.0"
+    runtimeOnly "org.postgresql:postgresql"
+    runtimeOnly "mysql:mysql-connector-java"
+    runtimeOnly "org.mariadb.jdbc:mariadb-java-client"
 
     testImplementation libraries.core_testing
     testImplementation libraries.liquibase_slf4j
     testImplementation libraries.jmock
 
     // Only include bouncycastle for testing purposes (compare against jss)
-    testImplementation "org.bouncycastle:bcpkix-jdk15on:1.60"
-    testImplementation "org.bouncycastle:bcprov-jdk15on:1.60"
+    testImplementation "org.bouncycastle:bcpkix-jdk15on"
+    testImplementation "org.bouncycastle:bcprov-jdk15on"
```

Comment:
Please remove these lines, otherwise changes look good to me.

Reply:
@kahowell Done.